eth/eventhandler/handlers.go

```diff
@@ -307,7 +307,7 @@ func (eh *EventHandler) validatorAddedEventToShare(
 	validatorShare.Quorum, validatorShare.PartialQuorum = ssvtypes.ComputeQuorumAndPartialQuorum(len(committee))
 	validatorShare.DomainType = eh.networkConfig.Domain
 	validatorShare.Committee = committee
-	validatorShare.Graffiti = []byte("ssv.network")
+	validatorShare.Graffiti = eh.graffiti
```

Comment:
this will not update existing shares in DB

i think we're gonna have to instead use graffiti passed down from config instead of from shares in the ProposerRunner